src/motor_control/src/motor_control_node.cpp

```diff
@@ -218,11 +224,12 @@ class MotorControlNode : public rclcpp::Node {
     // Define default values for our ROS parameters below #
     this->declare_parameter("CAN_INTERFACE_TRANSMIT", "can0");
     this->declare_parameter("CAN_INTERFACE_RECEIVE", "can0");
-    this->declare_parameter("BACK_LEFT_TURN", 2);
-    this->declare_parameter("FRONT_LEFT_TURN", 4);
+    this->declare_parameter("BACK_LEFT_TURN", 4);

```

Comment:
Are default values needed, if not just leave them und set. If you leave this you'll need to change this value in multiple places

Reply:
Yes, default values are needed. This is what is used when you "ros2 run" the node (no launch file) or when you launch the node without specifying a config file for the node.

Reply:
Is that a good behavior though? I definitely see a see a scenario where you think you are updating a value but you aren't correctly updating it. If you see that as acceptable, then keep it, but looks like this mechanism could work, allow_undeclared_parameters

Reply:
it's fine for now